Cmake Error on Ubuntu 20.04

Trying to install from source on Ubuntu 20.04 and getting this (partial log)...
```
Creating build directory...
Generating makefiles...
--  CMakeLists: Sonic Pi
-- The C compiler identification is GNU 9.4.0
-- The CXX compiler identification is GNU 9.4.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- TARGET_LINUX
-- System: Linux-5.13.0-52-generic
-- Compiler: GNU
-- Compiler Version: 9.4.0
-- Debug Flags: -g -D_DEBUG -DDEBUG
-- Release Flags: -O3 -DNDEBUG
--  Aubio Builder
-- ERLANG_INCLUDE_PATH: /home/bjorn/.asdf/installs/erlang/25.0.2/erts-13.0.2/include
-- USE_SYSTEM_LIBS: OFF
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Check if compiler accepts -pthread
-- Check if compiler accepts -pthread - yes
-- Found Threads: TRUE  
-- ImGui Interface: ON
--  CMakeLists: Sonic Pi
-- App Root: /home/bjorn/dev/sonic-pi/app
-- Configuring done
CMake Error at vcpkg/scripts/buildsystems/vcpkg.cmake:573 (_add_executable):
  Target "sonic-pi" links to target "Qt::PrintSupport" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?
Call Stack (most recent call first):
  gui/qt/CMakeLists.txt:203 (add_executable)


CMake Error at vcpkg/scripts/buildsystems/vcpkg.cmake:573 (_add_executable):
  Target "sonic-pi" links to target "Qt::Widgets" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?
Call Stack (most recent call first):
  gui/qt/CMakeLists.txt:203 (add_executable)


CMake Error at vcpkg/scripts/buildsystems/vcpkg.cmake:573 (_add_executable):
  Target "sonic-pi" links to target "Qt::OpenGL" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?
Call Stack (most recent call first):
  gui/qt/CMakeLists.txt:203 (add_executable)


CMake Error at vcpkg/scripts/buildsystems/vcpkg.cmake:573 (_add_executable):
  Target "sonic-pi" links to target "Qt::Xml" but the target was not found.
  Perhaps a find_package() call is missing for an IMPORTED target, or an
  ALIAS target is missing?
Call Stack (most recent call first):
  gui/qt/CMakeLists.txt:203 (add_executable)


CMake Error at vcpkg/scripts/buildsystems/vcpkg.cmake:573 (_add_executable):
  Target "sonic-pi" links to target "Qt::Core" but the target was not found.
  Perhaps a find_package() call is missing for an IMPORTED target, or an
  ALIAS target is missing?
Call Stack (most recent call first):
  gui/qt/CMakeLists.txt:203 (add_executable)


CMake Error at vcpkg/scripts/buildsystems/vcpkg.cmake:573 (_add_executable):
  Target "sonic-pi" links to target "Qt::Gui" but the target was not found.
  Perhaps a find_package() call is missing for an IMPORTED target, or an
  ALIAS target is missing?
Call Stack (most recent call first):
  gui/qt/CMakeLists.txt:203 (add_executable)


CMake Error at vcpkg/scripts/buildsystems/vcpkg.cmake:573 (_add_executable):
  Target "sonic-pi" links to target "Qt::Widgets" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?
Call Stack (most recent call first):
  gui/qt/CMakeLists.txt:203 (add_executable)


CMake Error at vcpkg/scripts/buildsystems/vcpkg.cmake:573 (_add_executable):
  Target "sonic-pi" links to target "Qt::OpenGL" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?
Call Stack (most recent call first):
  gui/qt/CMakeLists.txt:203 (add_executable)


CMake Error at vcpkg/scripts/buildsystems/vcpkg.cmake:573 (_add_executable):
  Target "sonic-pi" links to target "Qt::Concurrent" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?
Call Stack (most recent call first):
  gui/qt/CMakeLists.txt:203 (add_executable)


CMake Error at vcpkg/scripts/buildsystems/vcpkg.cmake:573 (_add_executable):
  Target "sonic-pi" links to target "Qt::Network" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?
Call Stack (most recent call first):
  gui/qt/CMakeLists.txt:203 (add_executable)


CMake Error at vcpkg/scripts/buildsystems/vcpkg.cmake:613 (_add_library):
  Target "QScintilla" links to target "Qt::PrintSupport" but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?
Call Stack (most recent call first):
  gui/qt/QScintilla_src-2.13.3/CMakeLists.txt:283 (add_library)


CMake Error at vcpkg/scripts/buildsystems/vcpkg.cmake:613 (_add_library):
  Target "QScintilla" links to target "Qt::Widgets" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?
Call Stack (most recent call first):
  gui/qt/QScintilla_src-2.13.3/CMakeLists.txt:283 (add_library)


CMake Error at vcpkg/scripts/buildsystems/vcpkg.cmake:613 (_add_library):
  Target "QScintilla" links to target "Qt::OpenGL" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?
Call Stack (most recent call first):
  gui/qt/QScintilla_src-2.13.3/CMakeLists.txt:283 (add_library)


CMake Error at vcpkg/scripts/buildsystems/vcpkg.cmake:613 (_add_library):
  Target "QScintilla" links to target "Qt::Xml" but the target was not found.
  Perhaps a find_package() call is missing for an IMPORTED target, or an
  ALIAS target is missing?
Call Stack (most recent call first):
  gui/qt/QScintilla_src-2.13.3/CMakeLists.txt:283 (add_library)


-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.
```

I got the same errors. 
I did install according to the instructions.
anders@thisbe:~/bin/sonic-pi/app$ qmake -query "QT_INSTALL_PREFIX"
/usr

But after upgrading to latest cmake from 3.18.4 to 3.23.2 I got fewer errors:
Generating makefiles...
--  CMakeLists: Sonic Pi
-- TARGET_LINUX
-- System: Linux-5.13.0-51-generic
-- Compiler: GNU
-- Compiler Version: 9.4.0
-- Debug Flags: -g -D_DEBUG -DDEBUG
-- Release Flags: -O3 -DNDEBUG
--  Aubio Builder
-- ERLANG_INCLUDE_PATH: /usr/lib/erlang/erts-13.0.2/include
-- USE_SYSTEM_LIBS: OFF
-- ImGui Interface: ON
--  CMakeLists: Sonic Pi
-- App Root: /home/anders/bin/sonic-pi/app
-- Qtapp Root: /home/anders/bin/sonic-pi/app/gui/qt
-- Configuring done
CMake Error at gui/qt/CMakeLists.txt:288 (target_link_libraries):
  Target "sonic-pi" links to:

    Qt::Core

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.



CMake Error at gui/qt/QScintilla_src-2.13.3/CMakeLists.txt:293 (target_link_libraries):
  Target "QScintilla" links to:

    Qt::PrintSupport

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.



-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.
---------------------------------------------

--- a/app/gui/qt/CMakeLists.txt
+++ b/app/gui/qt/CMakeLists.txt
@@ -288,24 +288,39 @@ target_link_libraries(${APP_NAME}
     PRIVATE
     SonicPi::API
     QScintilla
-    Qt::Core
-    Qt::Gui
-    Qt::Widgets
-    Qt::OpenGL
-    Qt::Concurrent
-    Qt::Network
     Threads::Threads)
 
-if(Qt6_FOUND)
+if(Qt5_FOUND)
   target_link_libraries(${APP_NAME}
   PRIVATE
+  Qt5::Core
+  Qt5::Gui
+  Qt5::Widgets
+  Qt5::OpenGL
+  Qt5::Concurrent
+  Qt5::Network)
+else()
+  target_link_libraries(${APP_NAME}
+  PRIVATE
+  Qt::Core
+  Qt::Gui
+  Qt::Widgets
+  Qt::OpenGL
+  Qt::Concurrent
+  Qt::Network
   Qt::OpenGLWidgets)
 endif()
 
 if(WITH_QT_GUI_WEBENGINE)
-  target_link_libraries(${APP_NAME}
-  PRIVATE
-  Qt::WebEngineWidgets)
+  if(Qt5_FOUND)
+    target_link_libraries(${APP_NAME}
+    PRIVATE
+    Qt5::WebEngineWidgets)
+  else()
+    target_link_libraries(${APP_NAME}
+    PRIVATE
+    Qt::WebEngineWidgets)
+  endif()
 endif()
 
 

--- a/app/gui/qt/QScintilla_src-2.13.3/CMakeLists.txt
+++ b/app/gui/qt/QScintilla_src-2.13.3/CMakeLists.txt
@@ -290,13 +290,23 @@ target_include_directories(QScintilla
   src
 )
 
-target_link_libraries(QScintilla
-PUBLIC
-Qt::PrintSupport
-Qt::Widgets
-Qt::OpenGL
-Qt::Xml
-)
+if(Qt5_FOUND)
+  target_link_libraries(QScintilla
+  PUBLIC
+  Qt5::PrintSupport
+  Qt5::Widgets
+  Qt5::OpenGL
+  Qt5::Xml
+  )
+else()
+  target_link_libraries(QScintilla
+  PUBLIC
+  Qt::PrintSupport
+  Qt::Widgets
+  Qt::OpenGL
+  Qt::Xml
+  )
+endif()
 
 target_compile_definitions(QScintilla PRIVATE -DSCINTILLA_QT)
 target_compile_definitions(QScintilla PRIVATE -DSCI_LEXER)
@@ -324,7 +334,7 @@ elseif(${CMAKE_SYSTEM_NAME} MATCHES Darwin) # macOS
     find_package(Qt5 COMPONENTS MacExtras REQUIRED)
     target_link_libraries(QScintilla
       PUBLIC
-      Qt::MacExtras
+      Qt5::MacExtras
       )
   endif()
 

